src/atoms/Text/Text.web.js

```diff
@@ -1,3 +1,140 @@
-const Text = () => null;
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import baseTheme from '../../tokens/theme';
+import { getColorKeys, getColor, getLineHeight } from '../../_helpers/theme';
+
+/*
+ * Sets height for the div
+ * @param {Object} theme
+ * @param {String} _lineHeight
+ * @param {number} maxLines
+ */
+const calculateHeight = ({ theme, _lineHeight, maxLines }) => {
+  const lineHeight = `${parseFloat(theme.fonts.lineHeight[_lineHeight])}`;
```

Comment:
lineHeight needs to be derived from size. Refer to the figma file. Line height is different for different font sizes